<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowLevelWidget</class>
 <widget class="QWidget" name="WindowLevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window/level</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="levelLabel">
          <property name="text">
           <string>Level</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="levelDoubleSpinBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="windowHorizontalSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="levelHorizontalSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="windowDoubleSpinBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="windowLabel">
          <property name="text">
           <string>Window</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="resetPushButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
